okta-spring-security-starter/src/main/java/com/okta/spring/oauth/OauthProperties.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2017 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.spring.oauth;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("okta.oauth")
+public class OauthProperties {
```

Comment:
At Stormpath, we renamed everything from `Oauth` to `OAuth`. Should we do the same here? Also, this might be better named `OktaProperties` or `OAuthOktaProperties`, unless it can be used by other OAuth providers.